### DISCUSSION TITLE
VPotRingLEDs with FastLED

### DISCUSSION THREAD

**Discussion by Benator7** [tttapa/Control-Surface#745] [Q&A]
Hey guys, first of all thx for this great library :)

I have a question regarding the VPotRing class.
Is it somehow possible to use this class (as bankable) with FastLED Library? Actually I'm trying to get some VPots working with ws8212 leds and a FOR loop to check for vpot[i].getStartOn() and vpot[i].getStartOff() and light the leds accordingly, but it only some of them will light up and flash randomly in all colors :D

**Comment by tttapa** [comment]
> I'm trying to get some VPots working with ws8212 leds and a FOR loop to check for vpot[i].getStartOn() and vpot[i].getStartOff() and light the leds accordingly,

Yes, that should work. Hard to say anything more concrete without seeing the code.

**Comment by Jhenesson** [comment]
Hi @Benator7 , I'm trying to get this to work, could you share the code?
